.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,45 @@
+name: pull-request
+
+on:
+- pull_request
+
+jobs:
+  unit-test:
+    strategy:
+      matrix:
+        go-version: [1.10.x, 1.11.x, 1.12.x, 1.13.x]
```

Comment:
doesn't an equivalent to `golang:latest` exist ?

Reply:
Maybe :sweat_smile: but IMHO relying on `latest` is a very bad idea since you don't know for sure which version it will be, you don't know if you have a cached version somewhere and it's not best practice :smile: the best, in my opinion, would be to also specify the patch version but I kept the latest patch in order to be coherent with Circle CI (and today latest points to 1.13, so it's already tested)